src/groupWith.js

```diff
@@ -8,23 +8,23 @@ var _curry2 = require('./internal/_curry2');
  * @memberOf R
  * @since v0.21.0
  * @category List
- * @sig (a, a -> Boolean) -> [a] -> [[a]]
+ * @sig ((a, a) → Boolean) → [a] → [[a]]
```

Comment:
@CrisFeo showed me this font package and mentioned that he might have some interest in contributing.